modules/qt/hm.nix

```
@@ -70,6 +72,9 @@
         )
         ++ (lib.optional (config.qt.style.name != recommendedStyle)
           "stylix: qt: Changing `config.qt.style` is unsupported and may result in breakage! Use with caution!"
+        )
+        ++ (lib.optional (config.qt.style.platformTheme.name == null)
+          "stylix: qt: Automatic Theme selection for Qt can only work when stylix is enabled system-wide. Set `stylix.enable = true` in `configuration.nix` .nix"
```

flake users don't necessarily have a `configuration.nix` 
```suggestion
          "stylix: qt: Automatic Theme selection for Qt can only work when stylix is enabled system-wide. Set `stylix.enable = true` in your NixOS configuration"
```

---

Can we ignore Darwin? Maybe "system configuration"